### DISCUSSION TITLE
Need Help with Student Developer Pack Application StSubject: Verified Student Developer Pack – Still No Access?atus

### DISCUSSION THREAD

**Discussion by Srikanth2304** [community/community#143116] [GitHub Education]
Hello GitHub Community!

I applied for the GitHub Student Developer Pack, and my application was successfully verified. However, I still don’t have access to the developer pack’s resources and benefits. I’ve checked my account, but nothing from the pack seems to be activated.

Has anyone experienced a similar delay after verification? If so, is there anything specific I should do or check to ensure access is granted? Any guidance would be much appreciated!

![image](https://github.com/user-attachments/assets/b0b145f5-dee3-4109-b0db-cd5954778844)

Thanks in advance for your help!

**Comment by davevad93** [comment]
Hi @Srikanth2304 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Srikanth2304 was approved and your benefits are now available.`

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Srikanth2304** [reply]
please make it fast as already , 3days was completed

**Reply by Srikanth2304** [reply]
Thank You, My benifits are available now.

**Reply by davevad93** [reply]
You're welcome @Srikanth2304 , have a nice day! 👍
